documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -94,6 +94,11 @@ Capacity limits can be set for the following broker resources, using the standar
 
 Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
 
+[NOTE]
+====
+We recommend that `Kafka.spec.kafka.resources.requests` and `Kafka.spec.kafka.resources.limits` are set to equal values. With these settings, broker pods will have the same capacity and quality of service is guaranteed. The settings can improve stability. This is especially important when using nodes with heterogeneous resources, such as CPU cores.
+====
+
```

Comment:
I don't think we recommend this. The only thing we recommend is to make sure that the Java memory of the Kafka brokers fits into the memory request.

Reply:
OK let me rephrase it.